mbq/metrics/utils.py

```diff
@@ -9,6 +9,9 @@ class MetricsError(Exception):
     pass
 
 
+class NoNameSpaceError(Exception):
```

Comment:
Looks like this isn't being used but if we're going to raise an exception I think it should just be a `ValueError`

Reply:
Ah yeah I hadn't decided whether I should raise an error if there's no namespace yet. Ok ValueError works